### PR TITLE
docs: add changelog and readme for Copilot integration

### PR DESCRIPTION
**Pull request by formulahendry**

### CODE REVIEW

File: PRERELEASE.md [hunk 1]
```diff
@@ -3,11 +3,17 @@
 
 > Note: This changelog only includes the changes for the pre-release versions of Azure API Center for Visual Studio Code. For the changelog of stable versions, please refer to the [Changelog of Azure API Center for Visual Studio Code](https://github.com/microsoft/vscode-azureapicenter/blob/main/CHANGELOG.md).
 
+### November 4, 2024
+* Add integration with GitHub Copilot for Azure (@azure extension) to provide a chat experience to let API developers use natural language to quickly generate OpenAPI specification compliant with Spectral rulesets:
```
Comment (juliajuju93) [inline PRERELEASE.md:7]: Add integration with GitHub Copilot for Azure (@azure extension) to provide a chat experience for developers to leverage its capabilities for a variety of API Center-related tasks:

File: PRERELEASE.md [hunk 2]
```diff
@@ -3,11 +3,17 @@
...
+    * Generate OpenAPI specification compliant with active API style guide.
```
Comment (juliajuju93) [inline PRERELEASE.md:8]: * Generate OpenAPI specification compliant with the active API style guide.

File: PRERELEASE.md [hunk 3]
```diff
@@ -3,11 +3,17 @@
...
+    * Generate OpenAPI specification compliant with default 'spectral:oas' ruleset, if no active API style guide is set.
```
Comment (juliajuju93) [inline PRERELEASE.md:9]: *  If no active API style guide is set, generate OpenAPI specifications compliant with the default ‘spectral:oas’ ruleset.
*  Easily register your generated APIs in API Center.

File: README.md [hunk 4]
```diff
@@ -22,9 +22,13 @@ Ensure that new API versions don't break API consumers with breaking change dete
 
 ![Breaking Change Detection](./media/breaking-change.png)
 
-[[Pre-Release]](#pre-release) Generate Open API specification file from API code.
+[[Pre-Release]](#pre-release) Use natural language to generate OpenAPI specification compliant with Spectral rulesets.
```
Comment (juliajuju93) [inline README.md:25]: Use GitHub Copilot for Azure to design API specifications that comply with API Center governance, ensuring your APIs adhere to best practices and standards through AI assistance.

**Review by adashen — APPROVED** [review]

**Review by wenytang-ms — APPROVED** [review]

**Pull request merged**